File: class17/cpp/AP1710-MenuCalculadora.cpp
#include <iostream>
#include <iomanip>
int main() {
    int op; double a, b;
    std::cout << "=== CALCULADORA ===" << std::endl;
    std::cout << "1 - Somar\n2 - Subtrair\n3 - Multiplicar\n4 - Dividir" << std::endl;
    std::cout << "Escolha: "; std::cin >> op;
    std::cout << "Número 1: "; std::cin >> a;
    std::cout << "Número 2: "; std::cin >> b;
    switch (op) {
        case 1: std::cout << a << " + " << b << " = " << a+b << std::endl; break;
        case 2: std::cout << a << " - " << b << " = " << a-b << std::endl; break;
        case 3: std::cout << a << " × " << b << " = " << a*b << std::endl; break;
        case 4:
            if (b!=0) std::cout << a << " ÷ " << b << " = " << std::fixed << std::setprecision(2) << a/b << std::endl;
            else std::cout << "Erro: divisão por zero!" << std::endl;
            break;
        default: std::cout << "Opção inválida!" << std::endl;
    }
    return 0;
}

File: class17/python/AP1710-MenuCalculadora.py
# MenuCalculadora — Aula 17
# Sistema de menu com 4 operações usando escolha

print("=== CALCULADORA ===")
print("1 - Somar")
print("2 - Subtrair")
print("3 - Multiplicar")
print("4 - Dividir")
op = int(input("Escolha: "))
a = float(input("Número 1: "))
b = float(input("Número 2: "))
if op == 1: print(f"{a} + {b} = {a+b}")
elif op == 2: print(f"{a} - {b} = {a-b}")
elif op == 3: print(f"{a} × {b} = {a*b}")
elif op == 4:
    if b != 0: print(f"{a} ÷ {b} = {a/b:.2f}")
    else: print("Erro: divisão por zero!")
else: print("Opção inválida!")
